app/polymer/src/server/config/defaults/webpack.config.js

```diff
@@ -1,73 +1,5 @@
-// import webpack from 'webpack';
-import autoprefixer from 'autoprefixer';
+import { createDefaultWebpackConfig } from '@storybook/core/server';
```

Comment:
How about making the webpack configs a separate package?

So this becomes much easier:

Reply:
Also, this would be a breaking change, can we make it so it's backwards compatible ?

Reply:
Actually, in order to be backward compatible, I left these files in every app =) So why is it a breaking change?

Reply:
What do you mean? Will it be something like `@storubook/configs` ? TBH, I think we need to make some progress in core extraction because this copypasta across all the apps makes me crazy 😷

Reply:
`@storybook/webpack-config` sounds good to me

Reply:
But maybe it would be an even better solution just to pass genDefaultConfig as third argument to custom webpack config function, so that users could write full control + defaults like this:

```js
module.exports = (baseConfig, env, genDefaultConfig) => {
  const config = genDefaultConfig(baseConfig, env);

  // Extend it as you need.
};
```

Reply:
Makes sense to me. It's way much simpler than publishing one more package

Reply:
I mean from the user's point of view =)

Reply:
I'd rather not create yet another package. @Hypnosphi the proposals above seem pretty reasonable--are they backwards compatible or breaking changes?

Reply:
@shilman We should deprecate the old way using `utils-deprecate` and remove it in next major